### ISSUE TITLE
differentiate between math operators

### ISSUE THREAD

**Issue by lorenzwalthert** [krlmlr/styler#86]
Allow the user to set spacing separately for 
* \+ and -
* / and *
* ^

etc.

**Comment by krlmlr** [comment]
Reference: https://github.com/krlmlr/styler/pull/79#discussion_r129096510

**Comment by lorenzwalthert** [comment]
Should this be a separate option or go under strict? I suggest creating a new argument `math_token_spacing` that takes a list that has two lists in it: One containing the tokens where we want `1` space before/after it and one that contains the tokens that should have no spacing before/after them. We can use a helper function to create this list like that
```r
specify_math_token_spacing(zero = c("'*'", "'/'"), one = c(...)
```
Tokens that are not mentioned in `zero` or `one` will have space around it, since that the tidyverse style guide encourages.

**Comment by lorenzwalthert** [comment]
Note that the proposed API is similar to the one proposed in #217 where we have
`reindention = specify_reindention(...)`

**Comment by lorenzwalthert** [comment]
Closed with #221.
